android-components/samples/browser/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature
```

Comment:
Should this be in the library module that users these hardware features instead?

Reply:
I originally thought so myself, but I preferred to add it to the app manifests as the permission was declared directly in the manifest, not in the merged one; so for visibility sake.